app/helpers/helper.rb

```diff
@@ -0,0 +1,11 @@
+helpers do
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:id])
```

Comment:
is it session[:id] or session[:user_id]? I have been seeing examples with the later.